scipy/stats/stats.py

```diff
@@ -1229,10 +1229,9 @@ def skew(a, axis=0, bias=True, nan_policy='propagate'):
 
     m2 = moment(a, 2, axis)
     m3 = moment(a, 3, axis)
-    zero = (m2 == 0)
-    vals = _lazywhere(~zero, (m2, m3),
-                      lambda m2, m3: m3 / m2**1.5,
-                      0.)
+    with np.errstate(all='ignore'):
+        zero = (np.abs(m2) <= (np.finfo(m2.dtype).resolution * a.mean(axis))**2)
+        vals = np.where(zero, 0, m3 / m2**1.5)
```

Comment:
Changed from `_lazywhere` to keep things consistent? (Should these be `_lazywhere` everywhere?)

Reply:
Consistency and also performance. `_lazywhere` is generally slower than simple numpy calls. e.g. for a single 1000 element array, `skew` was 50 us (~40%) slower than `kurtosis` despite being almost identical.

Reply:
Sounds good.